From ee3a4c7b0f8a69f0415f31c836dafe7d9309db78 Mon Sep 17 00:00:00 2001
From: Felipe Balbi <balbi@ti.com>
Date: Tue, 26 Nov 2013 10:06:00 -0600
Subject: [PATCH] usb: phy: fsm: don't depend on indirect includes

this header uses spinlocks and errno values, so
we must include <linux/spinlock.h> and <linux/errno.h>
to avoid build errors.

Signed-off-by: Felipe Balbi <balbi@ti.com>
(cherry picked from commit cc9d9ccf2a1e8c9f36a436805317180003ba9719)
Signed-off-by: Maladau Mou <maladau.mou@intel.com>
---
 include/linux/usb/otg-fsm.h |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/usb/otg-fsm.h b/include/linux/usb/otg-fsm.h
index 7441b46..32f86a3 100644
--- a/include/linux/usb/otg-fsm.h
+++ b/include/linux/usb/otg-fsm.h
@@ -15,6 +15,9 @@
  * 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
+#include <linux/spinlock.h>
+#include <linux/errno.h>
+
 #undef VERBOSE
 
 #ifdef VERBOSE
-- 
1.7.9.5

